packages/terra-progress-bar/src/terra-dev-site/doc/example/ProgressBarTwoColors.jsx

```diff
@@ -3,10 +3,14 @@ import ProgressBar from 'terra-progress-bar';
 import classNames from 'classnames/bind';
 import styles from './colors.module.scss';
 
+// eslint-disable-next-line import/order
+import StylesDoc from 'terra-progress-bar/docs/twoColors.doc.scss?dev-site-codeblock';
+
 const cx = classNames.bind(styles);
 
 const ProgressBarGradient = () => (
   <div>
+    <StylesDoc />
```

Comment:
@pranav300 This is a clever idea, but I don't think it gets us far enough. I'm going to look into automatically adding css to our example template in terra dev site. For now remove showing css in the examples.

Reply:
updated here https://github.com/cerner/terra-core/pull/2863/commits/59b1c324660bfc9b064c46391bdeb2398df0cf62.